Aboutme page

Liliya-V opened the pull request:
Home and About me navbar lines are now clicable.
Social media and navbar menu icons have a hover color.
The body size is set relatively to a viewport so it's not scrollable.
Responsive to a widht and height of a screen.

Liliya-V committed:
created an aboutme.html, copy pasted a template page code into it

Liliya-V committed:
added about me text
adjusted HTML

Liliya-V committed:
added style to aboutme text (class aboutme)

Liliya-V committed:
Revert "added style to aboutme text (class aboutme)"

This reverts commit 6b76910a356c83443183e0bc89317f6253aeacf8.

Liliya-V committed:
adjusted CSS so the style of aboutme page doesnt affect other branches

Liliya-V committed:
linked aboutme.html and homepage.html to its buttons in the navbar

Liliya-V committed:
Merge branch 'master' into aboutme

Liliya-V committed:
fixing css and image links

Liliya-V committed:
changed a hometop element color,
changes a navbar and a mypicture border color and hover color,
change a social element and its hover color

neilbulloch commented on aboutme.html:
```diff
@@ -0,0 +1,53 @@
+                I wish i was a developer...but i am not.
+            </h3>
+            <h4>
+                <p>
```
Why not have p tag on its own, rather than nested inside an h4? We can then style it with CSS to have the font-size and weight that we want.

Liliya-V replied:
I was thinking of using more p tags on other pages so have to use h to CSS them separetely.

Liliya-V committed:
added links to social media bar

mporam commented on aboutme.html:
```diff
@@ -0,0 +1,53 @@
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Portfolio Liliya Voevodina.html</title>
```
remove .html

mporam commented on aboutme.html:
```diff
@@ -0,0 +1,53 @@
+        </div>
+        <div class="homecenter">
+            <img class="aboutme-arrows" src="images/arrow-img.png" alt="Arrows">
+            <div class="aboutme-social">
```
why do you have different classes for the about me page? this should be the same as the homepage?

mporam commented on css/stylesheet.css:
```diff
@@ -107,4 +107,82 @@ h2 {
 .social a:hover {
     color: #adddef;
 }
+.aboutme-arrows {
```
this is the same as .arrows above, if you need to make changes, add 2 classes rather than duplicate code

mporam replied:
resolved

mporam commented on css/stylesheet.css:
```diff
@@ -107,4 +107,82 @@ h2 {
     color: #adddef;
 }
+.aboutme-arrows {
+    /*margin: auto auto;*/
```
remove comments

mporam commented on css/stylesheet.css:
```diff
@@ -107,4 +107,82 @@ h2 {
+    font-family: "NoeText", sans-serif;
+    font-size: 4vh;
+    z-index: 3;
+    /*position: absolute;*/
```
remove comments

mporam commented on css/stylesheet.css:
```diff
@@ -107,4 +107,82 @@ h2 {
+    font-family: "NoeText", sans-serif;
+    font-size: 2.5vh;
+    z-index: 3;
+    /*position: absolute;*/
```
remove comments

mporam commented on css/stylesheet.css:
```diff
@@ -107,4 +107,82 @@ h2 {
+
+.aboutme {
+    width: 30%;
+    /*margin: 0 auto;*/
```
remove comments

mporam reviewed (changes requested):
also look at coding standards. remmeber to layout your CSS correctly like last time

Liliya-V committed:
removed commented lines.
Removed HTML from the title.
Removed duplicated code, added multiple classes to the element instead.
Removed empty lines, tidied CSS

Liliya-V committed:
removed one space in HTML
